README.md

```diff
@@ -1,24 +1,176 @@
-# README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<a name="readme-top"></a>
 
-Things you may want to cover:
+# 📗 Table of Contents
 
-* Ruby version
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [📷 Video Recording](#video-record)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# 📖 DOC BOOK app  <a name="about-project"></a>
 
-* Database creation
+## 🛠 Built With <a name="built-with"></a>
 
-* Database initialization
+### Tech Stack <a name="tech-stack"></a>
 
-* How to run the test suite
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Ruby</li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby On Rails</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Video recording -->
+
+## 🚀 Live Demo <a name="video-record"></a>
+
+- [Video recording ]()
```

Comment:
### [OPTIONAL]

Dear @Christelle-12 
You should delete the Video Recording section if you decide not to record the video presentation of the application. If project requirements say you should record, then you should put the link.